lib/backbeat/routes.js

```diff
@@ -0,0 +1,35 @@
+/*
+    This file contains Backbeat API routes and route details
+*/
+
+/**
+ * The metrics route model.
+ * @return {Array} The array of route objects
+ */
+function routes() {
+    return [
+        // Route: /_/crr/failed?site=<site>&marker=<marker>
```

Comment:
minor: `sitename` query param

Reply:
Thanks, updated when I moved them back to Backbeat.